api/core/model_runtime/model_providers/tongyi/_common.py

```diff
@@ -1,4 +1,11 @@
-from core.model_runtime.errors.invoke import InvokeError
```

Comment:
these modifications look like not related to this PR, you can create another one instead.